src/servers/src/http/dashboard.rs

```diff
@@ -27,18 +28,32 @@ use crate::error::{BuildHttpResponseSnafu, Result};
 pub struct Assets;
 
 pub(crate) fn dashboard() -> Router {
-    Router::new().fallback(static_handler)
+    Router::new()
+        .route("/", routing::get(static_handler).post(static_handler))
+        .route("/*x", routing::get(static_handler).post(static_handler))
```

Comment:
What does `/*x` mean?

Reply:
It means all remaining parts a assigned to variable `x`. For instance, in `/book/12/name`, `x` is `book/12/name`